canvas_modules/harness/cypress/e2e/properties/expression-control.cy.js

```diff
@@ -23,33 +23,32 @@ describe("Test of expression editor control", function() {
 
 	it("Test syntax highlighting and autocomplete features in expression editor control", function() {
 		// syntax highlighting and autocomplete features
-		cy.verifyTypeOfWordInExpressionEditor("is_real", "keyword", "conditionExpr");
-		cy.verifyTypeOfWordInExpressionEditor("salbegin", "variable", "conditionExpr");
-		cy.verifyTypeOfWordInExpressionEditor("=", "operator", "conditionExpr");
-		cy.verifyTypeOfWordInExpressionEditor("120", "number", "conditionExpr");
-		cy.verifyTypeOfWordInExpressionEditor("F", "string", "conditionExpr");
+		cy.verifyTypeOfWordInExpressionEditor("is_real", ".ͼb", "conditionExpr");
+		cy.verifyVariablesAndOperatorsInExpressionEditor("salbegin", "conditionExpr");
+		cy.verifyVariablesAndOperatorsInExpressionEditor("=", "conditionExpr");
+		cy.verifyTypeOfWordInExpressionEditor("120", ".ͼd", "conditionExpr");
+		cy.verifyTypeOfWordInExpressionEditor("F", ".ͼe", "conditionExpr");
```

Comment:
Classes like `.ͼb, .ͼd, .ͼe, .ͼj, .ͼm` are internal codemirror 6 classes. They might change/break in newer CM 6 versions. Ref - https://discuss.codemirror.net/t/why-does-cm6-use-the-character-in-class-names/2821/2

Adding these unstable classes in cypress test because keywords, strings, comments appear in DOM like this -
![Screenshot 2024-02-07 at 5 11 13 PM](https://github.com/elyra-ai/canvas/assets/25124000/576c8ad9-e90e-4010-949d-1cb30b31ffdb)